lib/scholar/metrics/classification.ex

```diff
@@ -677,7 +737,119 @@ defmodule Scholar.Metrics.Classification do
     end
   end
 
-  defnp precision_recall_fscore_n(y_true, y_pred, beta, num_classes, average) do
+  @doc """
+  Calculates precision, recall, F-score and support for each
+  class. It also supports a `beta` argument which weights
+  recall more than precision by it's value.
+
+  ## Options
+
+  #{NimbleOptions.docs(@precision_recall_fscore_support_schema)}
+
+  ## Examples
+
+      iex> y_true = Nx.tensor([0, 1, 1, 1, 1, 0, 2, 1, 0, 1], type: :u32)
+      iex> y_pred = Nx.tensor([0, 2, 1, 1, 2, 2, 2, 0, 0, 1], type: :u32)
+      iex> Scholar.Metrics.Classification.precision_recall_fscore_support(y_true, y_pred, num_classes: 3)
+      {#Nx.Tensor<
```

Comment:
Doctests seem to be failing.

Reply:
When passing a tuple with tensors as a result you need to pass it as `Nx.tensor([...])` not with syntax printed in the terminal

Reply:
Not sure I follow, I've tried both this way:
```
iex> Scholar.Metrics.Classification.precision_recall_fscore_support(y_true, y_pred, num_classes: 3)
{Nx.Tensor([0.6666666865348816, 1.0, 0.25]),
Nx.Tensor([0.6666666865348816, 0.5, 1.0]),
Nx.Tensor([0.6666666865348816, 0.6666666865348816, 0.4000000059604645]),
Nx.Tensor([3, 6, 1])}
```

and also:
```
iex> Scholar.Metrics.Classification.precision_recall_fscore_support(y_true, y_pred, num_classes: 3)
{#Nx.Tensor<f32[3][0.6666666865348816, 1.0, 0.25]>, #Nx.Tensor<f32[3][0.6666666865348816, 0.5, 1.0]>, #Nx.Tensor<f32[3][0.6666666865348816, 0.6666666865348816, 0.4000000059604645]>, #Nx.Tensor<u64[3][3, 6, 1]>}
```

But both don't work, with the following given the following:
```
If you are planning to assert on the result of an iex> expression which contains a value inspected as #Name<...>, please make sure the inspected value is placed at the beginning of the expression; otherwise Elixir will treat it as a comment due to the leading sign #.
```

Reply:
Apologies, I didn't understand it. I've made a commit with the changes.